crypto/cipher/cipher_test.cc

```
@@ -229,46 +262,78 @@ static void TestCipherAPI(const EVP_CIPHER *cipher, Operation op, bool padding,
+        len = static_cast<int>(len_sz);
+        break;
       }
-    } else {
-      ASSERT_TRUE(EVP_CipherUpdate(ctx.get(), result.data() + total, &len,
-                                   in.data(), static_cast<int>(todo)));
     }
     ASSERT_GE(len, 0);
     total += static_cast<size_t>(len);
     in = in.subspan(todo);
   }
   if (op == Operation::kInvalidDecrypt) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)